cnxarchive/views.py

```diff
@@ -115,7 +115,7 @@ def is_latest(id, version):
 
 
 TYPE_INFO = []
-LEGACY_EXTENSION_MAP = {'epub': 'epub', 'pdf': 'pdf', 'zip': 'complete.zip'}
+LEGACY_EXTENSION_MAP = {'epub': 'epub', 'pdf': 'pdf', 'zip': 'offline.zip'}
```

Comment:
I apologize for including the change for #420 in this PR. It should be handled in a separate PR. I reverted the change with https://github.com/Connexions/cnx-archive/pull/477/commits/e6743ba54efb8bb900947541bc3367835284a31b.